components/61-layouts/detail-layout/_detail-layout.scss

```diff
@@ -166,7 +166,7 @@ aside.top {
 
       @include tablet {
         position: absolute;
-        top: -($bulb-image-width/2);
+        top: -($bulb-image-width*.5);
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.